_episodes/04-docker-hub.md

```diff
@@ -36,6 +47,91 @@ The main body of the page contains many used headings, such as:
 
 At least in my experience, the "Examples of how to use the image" section of most images' pages will provide examples that are likely to adequately cover your intended use of the image.
 
+## Exploring Image Versions
+
+A single Docker Hub page can have many different versions of container images,
+based on the version of the software inside.  These
+versions are indicated by "tags". When referring to the specific version of a container
+by its tag, you use a colon, `:`, like this:
+
+```
+CONTAINERNAME:TAG
+```
+
+So if I wanted to download the `python` container, with Python 3.8, I would use this name:
+
+```
+$ docker pull python:3.8
+```
+{: .bash}
+
+But if I wanted to download a Python 3.6 container, I would use this name:
+
+```
+$ docker pull python:3.6
+```
+{: .bash}
+
+The default tag (which is used if you don't specify one) is called `latest`.
+
+So far, we've only seen containers that are maintained by the Docker team. However,
+it's equally common to use containers that have been produced by individual owners
+or organizations. The most useful R containers are developed by an organization called
```

Comment:
The R comment seems a bit disconnected, maybe add into a callout instead?

Reply:
or maybe putting this in the paragraph with the challenge?

Reply:
sounds good, will tweak.